src/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureSVN.java

```diff
@@ -493,7 +493,16 @@ private boolean findSourceCodeSVN(HttpMessage originalMessage) throws Exception
 								}
 							}
 							catch (Exception e) {
-								log.error ("Error executing SQL on temporary SVN SQLite database '"+ sqliteConnectionUrl + "': "+ e);
+								StringBuilder errorSb=new StringBuilder(300);
+								errorSb.append("Error executing SQL on temporary SVN SQLite database '");
+								errorSb.append(sqliteConnectionUrl);
+								errorSb.append("'");
+								if (log.isDebugEnabled()) {
```

Comment:
I thought you were going to change only the log level (but rereading the issue comment you did mention this change.)
I think it's fine to log always the exception message, if it's debug then more information does not hurt.
(Technically the exception message will always be included, since the message will only be logged if debug is enabled, unless it's to log the exception stack trace?)